### PR TITLE
pipfile 0.0.2

### PR DESCRIPTION
**Pull request by holgern**
https://github.com/pypa/pipfile
https://pypi.python.org/pypi/pipfile

### CODE REVIEW

File: recipes/pipfile/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - python
+    - pip
+
+  run:
+    - python
+    - toml
+
+test:
+  imports:
+    - pipfile
```
Comment (CJ-Wright) [inline recipes/pipfile/meta.yaml]: Could you also add an import test for the api?

Comment (holgern) [reply recipes/pipfile/meta.yaml]: Ok, I added a import for pipfile.api.

**Comment by holgern** [comment]
https://github.com/conda-forge/pipfile-feedstock is still empty?
Do I have to wait, or was something going wrong....

**Comment by jakirkham** [comment]
It takes some time for things to be converted. Looks fine now.

Side note: Any thoughts on adding `pipenv`?

**Pull request merged**